.github/workflows/nodejs-demos.yml

```diff
@@ -38,6 +38,16 @@ jobs:
       uses: actions/setup-node@v3
       with:
         node-version: ${{ matrix.node-version }}
+    
+    # ************** REMOVE AFTER RELEASE ********************
+    - name: Pre-build binding dependencies
+      run: npm install yarn
+      working-directory: binding/nodejs
```

Comment:
does this actually do anything meaningful? I've been removing them because the builders all come with yarn, so it's just a wasted action